ConcaveFi/concave-frontend
# Ce 715 js delivr link conversion

## Description

Converted images hosted on github to use JSdelivr link to retrieve images faster
## Steps to UI Test

1.go to switch networks and verify logo are loading.
2.go to swap and select token, verify logo for tokens are loading.
3. add cnv to wallet and verify token logo is loading.
4. go to your pool and try to create new pool, verify tokens logo are loading.
5. go to stake user stake dashboard and verify logo are loading
6. go to treasury and verify token logos are loading.
7. go to marketplace and verify token logo are loading
8. go to stake page and verify token logos are loading.
## Checklist

- [x] PR is named correctly
  - example: `CE-123: description`, `CE123 description`, `description`
